Additional info to HelpMessage Parameter Attribute RE interaction with Comment Based Help (#6163)
MicrosoftDocs/PowerShell-Docs: reference/7.0/Microsoft.PowerShell.Core/About/about_Functions_Advanced_Parameters.md

argument has no effect on optional parameters.

The following example declares a mandatory **ComputerName** parameter and a
help message that explains the expected parameter value. 

> If there is no other [comment_based_help](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_comment_based_help?view=powershell-7) syntax
> for the function (e.g. .SYNOPSIS) then this message will also show up under Get-Help.

Reviewer: ```suggestion
If there is no other [comment-based help](./about_comment_based_help.md) syntax
for the function (for example, `.SYNOPSIS`) then this message also shows up in
`Get-Help` output.

```
